networking/multiple_networks/configuring-additional-network.adoc

```diff
@@ -19,13 +19,13 @@ As a cluster administrator, you can configure an additional network for your clu
 [id="{context}_approaches-managing-additional-network"]
 == Approaches to managing an additional network
 
-You can manage the life cycle of an additional network by two approaches. Each approach is mutually exclusive and you can only use one approach for managing an additional network at a time. For either approach, the additional network is managed by a Container Network Interface (CNI) plugin that you configure.
+You can manage the lifecycle of an additional network in OpenShift by using one of two approaches: modifying the Cluster Network Operator (CNO) configuration or applying a YAML manifest. Each approach is mutually exclusive and you can only use one approach for managing an additional network at a time. For either approach, the additional network is managed by a Container Network Interface (CNI) plugin that you configure. The two different approaches are summarized here: 
 
-For an additional network, IP addresses are provisioned through an IP Address Management (IPAM) CNI plugin that you configure as part of the additional network. The IPAM plugin supports a variety of IP address assignment approaches including DHCP and static assignment.
+* Modifying the Cluster Network Operator (CNO) configuration: Configuring additional networks through CNO is only possible for cluster administrators. The CNO automatically creates and manages the `NetworkAttachmentDefinition` object.
 
-* Modify the Cluster Network Operator (CNO) configuration: The CNO automatically creates and manages the `NetworkAttachmentDefinition` object. In addition to managing the object lifecycle the CNO ensures a DHCP is available for an additional network that uses a DHCP assigned IP address.
+* Applying a YAML manifest: You can manage the additional network directly by creating an `NetworkAttachmentDefinition` object. Compared to modifying the CNO configuration this approach gives you more granular control and flexibility when it comes to configuration. For example creating an `NetworkAttachmentDefinition` object allows for the chaining of CNI plugins.
 
-* Applying a YAML manifest: You can manage the additional network directly by creating an `NetworkAttachmentDefinition` object. This approach allows for the chaining of CNI plugins.
+For an additional network, IP addresses are provisioned through an IP Address Management (IPAM) CNI plugin that you configure as part of the additional network. The IPAM plugin supports a variety of IP address assignment approaches including DHCP and static assignment. For additional networks requiring an IPAM of `type` equal to `dhcp`, a DHCP sever must be running on the nodes. See the "Dynamic IP address (DHCP) assignment configuration" section for guidance on how to trigger the deployment of the DHCP server by editing the CNO configuration. 
```

Comment:
wait up -- this definitely needs a bit of refinement, there's a common misunderstanding here...

DHCP IPAM CNI plugin operates with two components:

* The CNI plugin itself
* The DHCP IPAM CNI daemon -- which is specifically *not* a DHCP server. This DHCP IPAM CNI daemon is a listener for DHCP events.

The DHCP server component is left to be for customer environments that already have DHCP servers and the customer would like to integrate with the assignment from their existing DHCP. 

In cases where a DHCP server is unavailable, we typically advise to use Whereabouts IPAM CNI.

Both Whereabouts and DHCP IPAM CNI benefit from the use of the CNO usage pattern, as both require daemons to be launch (in the case of Whereabouts, it's a reconciler to reconciler stale IP address allocations). However, the docs otherwise also show how to create a shim whereby these optional components are launched on an opt-in basis.